package.json

```diff
@@ -54,6 +54,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-router-dom": "^5.1.7",
```

Comment:
Since we are not building, but directly deploying app to the heroku I would suggest moving that to "dependencies" instead of "devDependencies"

Reply:
Done